modules/Nncase.Modules.CPU/CodeGen/CPU/CSourceCompiler.cs

```diff
@@ -172,7 +172,7 @@ private string ArgumentsSpecific(string sourcePath, string outPath)
         "-DCMAKE_C_COMPILER=clang-cl -DCMAKE_CXX_COMPILER=clang-cl" : string.Empty;
 
 #if DEBUG
-        var config = "Release";
+        var config = "Debug";
```

Comment:
This will make debug build runnig slowly.

Reply:
reverted